u19_pipeline/imaging_rec.py

```diff
@@ -0,0 +1,94 @@
+import datajoint as dj
+
+from u19_pipeline import acquisition, subject, recording
+
+schema = dj.schema(dj.config['custom']['database.prefix'] + 'imaging_rec')
+
+
+@schema
+class Scan(dj.Computed):
+    definition = """
+    # General information of an imaging session
+    -> recording.Recording
+    ---
+    """
+    @property
+    def key_source(self):
+        return recording.Recording & {'recording_modality': 'imaging'}
+
+    def make(self, key):
+        self.insert1(key)
+
+
+@schema
+class ImagingSegmentation(dj.Computed):
```

Comment:
```suggestion
class ImagingProcessing(dj.Computed):
```

I would rename to match my suggestion for the `ephys` schema.